docker/Dockerfile.test

```diff
@@ -1,4 +1,4 @@
-FROM php:7.4-fpm
+FROM php:8.0.2-fpm
```

Comment:
Why not `php:8.1.1-fpm` ? (https://symfony.com/doc/current/setup.html#technical-requirements)